README.md

```diff
@@ -27,9 +27,89 @@ If you are interested in what drove the need for this checkout [the why section]
 
 1. If you do not already have it installed, install Cypress and add it to your package file: `npm i --save-dev cypress` or `yarn add -D cypress`
 1. Make sure you have a `cypress` folder containing Cypress tests (or create one by calling `cypress open`)
+1. Set the following config in your `cypress.config.js`
+
+   ```js
+   const { defineConfig } = require('cypress');
+   const cypressFirebasePlugin = require('cypress-firebase').plugin;
+   const admin = require('firebase-admin');
+
+   module.exports = defineConfig({
+     e2e: {
+       baseUrl: 'http://localhost:3000',
+       supportFile: 'cypress/support/e2e/index.js',
+       setupNodeEvents(on, config) {
+         cypressFirebasePlugin(on, config, admin);
+         // e2e testing node events setup code
+       },
+     },
+   });
+   ```
+
+1. Add the following your custom commands file (`cypress/support/e2e/commands.js`):
+
+   ```js
+   import firebase from 'firebase/app';
+   import 'firebase/auth';
+   import 'firebase/database';
+   import 'firebase/firestore';
+   import { attachCustomCommands } from 'cypress-firebase';
+
+   const fbConfig = {
+     // Your config from Firebase Console
+   };
+
+   firebase.initializeApp(fbConfig);
+
+   attachCustomCommands({ Cypress, cy, firebase });
+   ```
+
+   With [Firebase Web SDK version 9](https://firebase.google.com/docs/web/modular-upgrade)
+
+   ```js
+   import firebase from 'firebase/compat/app';
+   import 'firebase/compat/auth';
+   import 'firebase/compat/database';
+   import 'firebase/compat/firestore';
+   import { attachCustomCommands } from 'cypress-firebase';
+
+   const fbConfig = {
+     // Your config from Firebase Console
+   };
+
+   firebase.initializeApp(fbConfig);
+
+   attachCustomCommands({ Cypress, cy, firebase });
+   ```
+
+1. Make sure that you load the custom commands file in an `cypress/support/e2e/index.js` like so:
```

Comment:
`cypress/support/e2e/index.js` does it mandatory to create a new folder `e2e`.
After cypress v10 migration structure will be `cypress/support/e2e.ts{js}`
would you recommend creating a new `e2e` folder inside the support folder

Reply:
Good callout, when reading the Cypress docs I took that to mean that it was a folder by default - I'll keep it with the standard and switch it back to a file